### ISSUE TITLE
License & usage suggestion?

### ISSUE THREAD

**Issue by AngledLuffa** [tk1980/LargeMarginInSoftmax#1]
Do you have a license for the code, especially the `LargeMarginInSoftmaxLoss`?  Or perhaps a pip installable module, if that's an option you've considered at all.

Also, is the loss function a drop-in replacement for `CrossEntropyLoss`?  I would guess so, since it inherits from that module.

Thanks!
